tools/tosa_metadata.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+# SPDX-FileCopyrightText: Copyright 2025 Arm Limited and/or its affiliates <open-source-office@arm.com>
+
+"""Parsing of tosa.xml to tosa_metadata.json"""
```

Comment:
Since this doesn't have any Python specific dependencies please converted to JavaScript using `../source/xml.js`.